### PR TITLE
pupups: header image, fix title and fields

### PR DESCRIPTION
**Pull request by matallo**
close https://github.com/CartoDB/cartodb/issues/9701

-bumped cartodb.js with latest changes

available in ded03

screen shot 2016-09-28 at 15 53 11

CR @alonsogarciapablo

### CODE REVIEW

File: lib/assets/javascripts/cartodb3/mustache-templates/infowindows/infowindow_header_with_image.jst.mustache [hunk 1]
```diff
@@ -4,24 +4,23 @@
...
-            {{^value}}{{/value}}
-          {{/index}}
-        {{/content.fields}}
+        {{#content.fields.1}}
+          {{#value}}
+            <h4 class="CDB-infowindow-title">
+              <span>{{{ value }}}</span>
+            </h4>
+          {{/value}}
+          {{^value}}{{/value}}
```
Comment (alonsogarciapablo) [inline lib/assets/javascripts/cartodb3/mustache-templates/infowindows/infowindow_header_with_image.jst.mustache]: Do we need this?

File: lib/assets/javascripts/cartodb3/mustache-templates/infowindows/infowindow_header_with_image.jst.mustache [hunk 2]
```diff
@@ -4,24 +4,23 @@
...
+        {{/content.fields.1}}
+        {{^content.fields.1}}{{/content.fields.1}}
```
Comment (alonsogarciapablo) [inline lib/assets/javascripts/cartodb3/mustache-templates/infowindows/infowindow_header_with_image.jst.mustache]: Do we need this?

File: lib/assets/javascripts/cartodb3/mustache-templates/infowindows/infowindow_header_with_image.jst.mustache [hunk 3]
```diff
@@ -4,24 +4,23 @@
...
-          {{^index}}
-            {{#value}}
-              <h4 class="CDB-infowindow-title">
-                <span>{{{ value }}}</span>
-              </h4>
-            {{/value}}
-            {{^value}}{{/value}}
-          {{/index}}
-        {{/content.fields}}
+        {{#content.fields.1}}
```
Comment (alonsogarciapablo) [inline lib/assets/javascripts/cartodb3/mustache-templates/infowindows/infowindow_header_with_image.jst.mustache]: Didn't know about this... is this getting the first field?

Comment (matallo) [inline lib/assets/javascripts/cartodb3/mustache-templates/infowindows/infowindow_header_with_image.jst.mustache]: actually the second, the immediate after the first one, which is supposed to be the url

File: lib/assets/javascripts/cartodb3/mustache-templates/infowindows/infowindow_header_with_image.jst.mustache (hunk 1 above)
Comment (matallo) [inline lib/assets/javascripts/cartodb3/mustache-templates/infowindows/infowindow_header_with_image.jst.mustache]: I think we can get rid of it, just following the convention as I saw it elsewhere in that same file, happy to remove it

**Comment by matallo** [comment]
🚀

**Pull request merged**